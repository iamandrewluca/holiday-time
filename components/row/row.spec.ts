import { describe, it, expect } from '@jest/globals'
import { render } from '@testing-library/vue'
import Row from './row.component.vue'

describe('Row', () => {
  it('should render and match snapshot', () => {
    const { baseElement } = render(Row)
    expect(baseElement).toMatchSnapshot()
  })
})
