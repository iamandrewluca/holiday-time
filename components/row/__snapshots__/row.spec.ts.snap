// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row should render and match snapshot 1`] = `
<body>
  <div>
    <div />
  </div>
</body>
`;
